docs/download.md

```diff
@@ -1,32 +1,46 @@
+<style>
+.warning {
+  padding: 20px;
+  background-color: #3e475b;
+  color: white;
+  opacity: 1;
+  transition: opacity 0.6s;
+  margin-bottom: 15px;
+  border-left: 5px solid #bd5634;
+}
+</style>
+
 Download Chunky
 ===============
 
-Download options for [version @VERSION@][1]:
+Download options for [version 2.3.0][1] for Minecraft 1.13+:
```

Comment:
Could we update the variable to simplify future changes to the page instead of dropping it? :thinking:

Reply:
Changing the variables within version.properties causes changes across the whole site. I don't really want the old docs to link to 2.X .jars if I can help it as this would create confusion. At the time we discussed this it we did decide that hardcoding was the best option. I will take another look at this because I could just hardcode the 1.4.5 stuff and use the version.properties for newer versions.